contrib/chart/backstage/values.yaml

```diff
@@ -3,11 +3,11 @@
 # Declare variables to be passed into your templates.
 
 frontend:
-  enabled: false
+  enabled: true
   replicaCount: 1
   image:
-    repository: martinaif/backstage-k8s-demo-frontend
-    tag: test1
+    repository: citihub/backstage-frontend
```

Comment:
Let's keep this as it was, the intention is to replace the image anyway

Reply:
The version of frontend in `martinaif/backstage-k8s-demo-frontend:test1` is incompatible with the latest backend. Ref issue https://github.com/backstage/backstage/issues/8572#issuecomment-999892757.

Reply:
Are the current images straight up broken with this new configuration? `url` has been around for a very long time 😅

@martina-if is it low effort to bump the demo image up to a more recent release?

Reply:
If the timestamp on the backend image `20210423T1550` is correct then they're 8 months old.  Certainly the templating is not working in those images, hence my motivation to build against the latest release.

Reply:
Perhaps we could just make these required instead? Or a more placeholdery value like `<changeme>`?

Reply:
I also think think it should be a placeholder value or a requirement. DCO is also missing for some of the commits making the build red.

Reply:
The great thing about having Helm charts available in a project is the ability to get Backstage up and running with minimal effort. If up-to-date images aren't included in the charts, requiring a build process, then it will turn a lot of people off trying it.

Reply:
p.s. I'll fix the DCO later...